lib/views/after_auth_screens/events/create_event_form.dart

```diff
@@ -70,6 +85,26 @@ class CreateEventForm extends StatelessWidget {
               ),
             ),
           ),
+          // Row(
+          //   children: [
+          //     const Icon(
+          //       Icons.place,
+          //     ),
+          //     SizedBox(
+          //       width: SizeConfig.screenWidth! * 0.036,
+          //     ),
+          //     TextButton(
+          //       onPressed: () => navigationService.pushScreen(Routes.mapScreen),
+          //       child: Text(
+          //         AppLocalizations.of(context)!.strictTranslate('Add Location'),
+          //         style: Theme.of(context)
+          //             .textTheme
+          //             .headline5!
+          //             .copyWith(fontSize: 16),
+          //       ),
+          //     )
+          //   ],
+          // ),
```

Comment:
May I ask why is this commented out for now?

Reply:
ok i forgot to remove this.